lib/screens/v2/widget_instance/normal_header.ex

```diff
@@ -30,6 +30,14 @@ defmodule Screens.V2.WidgetInstance.NormalHeader do
     [:header]
   end
 
+  def audio_serialize(%__MODULE__{screen: %Screen{app_id: :gl_eink_v2}, text: text, icon: icon})
+      when icon in [:green_b, :green_c, :green_d, :green_e] do
+    "green_" <> branch = to_string(icon)
+    %{text: text, branch: branch}
+  end
+
+  def audio_serialize(%__MODULE__{text: text}), do: %{text: text}
```

Comment:
should we add the `:gl_eink_v2` pattern? not needed, but will just protect us from thinking it's implemented for bus eink